### PR TITLE
Shape and Text Shadows

### PR DESCRIPTION
**Pull request by adkenyon**
# Summary

I needed shapes with shadows. 

## Test Plan

Tested using the example app. The Text has a blue shadow and the heart has a red shadow. (On Android the heart shadow is _very_ faint)

### IOS Example App
Simulator Screen Shot - iPhone 11 Pro Max - 2019-12-05 at 13 50 37

### Android Example App
Screenshot_1575589154

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅    |
| Android |    ✅    |

## Checklist

- [X] I have tested this on a device and a simulator
- [X] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [X] I updated the typed files (TS and Flow)
- [X] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by osikes** [comment]
This is awesome !!

**Comment by adkenyon** [comment]
@Esemesek ^

File: ios/ARTShadow.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//
```
Comment (Esemesek) [inline ios/ARTShadow.h]: I feel like this comment shouldn't be here :P

Comment (adkenyon) [reply ios/ARTShadow.h]: Updated with the comment used in all other files

File: lib/types.js [hunk 2]
```diff
@@ -31,6 +31,13 @@ export type TransformProps = {
   },
 };
 
+export type ShadowProps = {
```
Comment (Esemesek) [inline lib/types.js]: Since I just merged the `typings` PR, can you also update the `index.d.ts`? You will need to merge the latest `master` branch.

**Comment by adkenyon** [comment]
@Esemesek typings are updated and the merge conflict is resolved.

**Comment by Esemesek** [comment]
Thank you for contributing <3

**Pull request merged**